app/src/main/java/cm/aptoide/pt/billing/external/ExternalBillingBinder.java

```diff
@@ -56,25 +60,34 @@
   private final Context context;
   private final ExternalBillingSerializer serializer;
   private final PaymentThrowableCodeMapper errorCodeFactory;
-  private final Billing billing;
+  private final PackageManager packageManager;
   private final CrashReport crashReport;
-  private final BillingIdResolver idResolver;
   private final int supportedApiVersion;
   private final BillingAnalytics analytics;
 
+  private Billing billing;
+  private String merchantName;
+
   public ExternalBillingBinder(Context context, ExternalBillingSerializer serializer,
-      PaymentThrowableCodeMapper errorCodeFactory, Billing billing, CrashReport crashReport,
-      BillingIdResolver idResolver, int apiVersion, BillingAnalytics analytics) {
+      PaymentThrowableCodeMapper errorCodeFactory, CrashReport crashReport, int apiVersion,
+      BillingAnalytics analytics, PackageManager packageManager) {
     this.context = context;
     this.serializer = serializer;
     this.errorCodeFactory = errorCodeFactory;
+    this.packageManager = packageManager;
     this.billing = billing;
```

Comment:
Variable not being assigned, but is instantiated in the onTransact

Reply:
Fixed.